code/modules/surgery/organs/organ_icon.dm

```diff
@@ -162,10 +162,13 @@ GLOBAL_LIST_EMPTY(limb_icon_cache)
 		new_icon_state = "[icon_name][gendered_icon ? "_f" : ""]"
 	else
 		if(gendered_icon)
-			if(dna.GetUIState(DNA_UI_GENDER))
-				gender = "f"
-			else
-				gender = "m"
+			switch(dna.GetUITriState(DNA_UI_GENDER))
+				if(0)
+					gender = "f"
+				if(1)
+					gender = "m"
+				else
+					gender = "f"
```

Comment:
```suggestion
					gender = pick("f", "m")
```

Reply:
![image](https://user-images.githubusercontent.com/69871346/94300214-ab91d700-ff36-11ea-9fcd-dc8bdc8c06b1.png)

Reply:
Diona - same sprites
Drask - same sprites
Grey - different torso (the female one has breasts), different groin (female one is wider)
Vox - same sprites
Machine - same sprites

By "pick"ing gender each time, different sprites (for Grey) might be chosen depending on the organ. 

This proc is a getter, as per @farie82 's earlier comments, I'm concerned about having this proc ask for the organ icon file names and getting something like:

- Asks for torso icon, plural gender, picks from "f" and "m", sprite grabbed is "torso_f"
- Asks for groin icon for the same person, plural gender, picks from "f" and "m", sprite grabbed is "groin_m"

Right now, it doesn't matter so much since many tri-gendered species have the same sprites for "f" and "m", but if someone re-sprites them to be different, it might become noticeable.

Thoughts?

Reply:
Since "f" is used as the default in line 162, I'm going to use "f".